src/components/ToDoComponent/SettingGoals/SettingGoalsComponents/Calendar.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, StyleSheet, ScrollView } from 'react-native';
+import DatePicker from 'react-native-date-picker';
+
+const DeadLineDatePicker = () => {
```

Comment:
Component name is DeadLineDatePicker, yet the name of file is Calendar.tsx?

Reply:
Noted, will change the filename too.

Reply:
Done changed the file name.